apis/extensions/v1alpha1/addon_types.go

```diff
@@ -92,9 +92,10 @@ type InstallableSpec struct {
 }
 
 type SelectorRequirement struct {
-	// The selector key. Valid values are KubeVersion, KubeGitVersion.
+	// The selector key. Valid values are KubeVersion, KubeGitVersion and KubeProvider.
 	// "KubeVersion" the semver expression of Kubernetes versions, i.e., v1.24.
 	// "KubeGitVersion" may contain distro. info., i.e., v1.24.4+eks.
+	// "KubeProvider" the Kubernetes provider, i.e., aws,gcp,azure,huaweiCloud,tencentCloud etc.
```

Comment:
update punctuation, have space after comma.

Reply:
Delighted to receive @nashtsai 's suggestions, and I will fix these issues in next pull request.